src/classes/partialset.cpp

```diff
@@ -509,12 +517,37 @@ bool PartialSet::deserialise(LineParser &parser, const CoreData &coreData)
     {
         for (auto typeJ = typeI; typeJ < nTypes; ++typeJ)
         {
-            if (!partials_[{typeI, typeJ}].deserialise(parser))
-                return false;
-            if (!boundPartials_[{typeI, typeJ}].deserialise(parser))
+            auto &part = partials_[{typeI, typeJ}];
+            auto &bound = boundPartials_[{typeI, typeJ}];
+            auto &unbound = unboundPartials_[{typeI, typeJ}];
+
+            if (parser.getArgsDelim(LineParser::Defaults) != LineParser::Success)
                 return false;
-            if (!unboundPartials_[{typeI, typeJ}].deserialise(parser))
+            part.setTag(parser.argsv(0));
+            bound.setTag(parser.argsv(1));
+            unbound.setTag(parser.argsv(2));
+
+            Messenger::warn("Tags: {} {} {}", parser.argsv(0), parser.argsv(1), parser.argsv(2));
+
+            if (parser.getArgsDelim(LineParser::Defaults) != LineParser::Success)
                 return false;
+            auto nPoints = parser.argi(0);
+            auto partError = parser.argb(1);
+            auto boundError = parser.argb(2);
+            auto unboundError = parser.argb(3);
+
+            for (auto n = 0; n < nPoints; ++n)
+            {
+                if (parser.getArgsDelim(LineParser::Defaults) != LineParser::Success)
+                    return false;
+                int argIndex = 0;
+                part.xAxis().push_back(parser.argd(argIndex));
+                bound.xAxis().push_back(parser.argd(argIndex));
+                unbound.xAxis().push_back(parser.argd(argIndex++));
+                argIndex = readDataPoint(argIndex, parser, part);
+                argIndex = readDataPoint(argIndex, parser, bound);
+                argIndex = readDataPoint(argIndex, parser, unbound);
```

Comment:
Part of me wants to convert the x-axis value first and re-use it for clarity:
```suggestion
                auto x = parser.argd(argIndex++);
                part.xAxis().push_back(x);
                bound.xAxis().push_back(x);
                unbound.xAxis().push_back(x);
                argIndex = readDataPoint(argIndex, parser, part);
                argIndex = readDataPoint(argIndex, parser, bound);
                argIndex = readDataPoint(argIndex, parser, unbound);
```

Reply:
That is clearer.